dev/src/dev.clj

```diff
@@ -307,3 +317,15 @@
      (mt/with-driver (:engine db#)
        (mt/with-db db#
          ~@body))))
+
+(defn set-log-level!
+  "Set logging level for a namespace you're in right now."
+  ([level]
+   (set-log-level! (ns-name *ns*) level))
+  ([ns level]
+   (mt/set-ns-log-level! ns level)))
```

Comment:
`mt/set-ns-log-level!` already has a 1-arity that does it for `*ns*`, this is basically doing the same thing that function does, if you really want it here just import it with Potemkin. We don't need 2 functions that do the exact same thing